src/components/islands/search.jsx

```diff
@@ -222,7 +222,7 @@ function SearchResult({ result = [], cursor, searchTerm }) {
   const router = useRouter();
 
   return (
-    <div className="block w-full text-sm md:text-base">
+    <div className="block w-full text-sm md:text-base  max-h-[505px] overflow-y-scroll">
```

Comment:
The maximum height (`max-h-**`) approach is great, exactly the thing that does it! But, I thought that the value of `505px` is a little rigid 🤔 

Take an instance where the viewport isn't even up to `505px` in size, the dialog would still enforce a `505px` hence shooting the div off the page... see screencast below 

[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2024.08.24-23_50_52.webm](https://github.com/user-attachments/assets/fbee595f-bcb7-4a12-90ef-85e559b7470a)

### Here's what I would do 😉 

- Transfer the max-width styling to the parent div of both the search form and search result div, setting it to `max-h-full` i.e. `max-height: 100%` making sure that the child respects the available width.

[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2024.08.24-23_58_26.webm](https://github.com/user-attachments/assets/7e2dba2e-3a52-474b-8a3b-e0cbde3321da)